ddl.lua

```diff
@@ -39,11 +42,44 @@ local function check_schema_format(schema)
 end
 
 local function _check_schema(schema)
+    -- Create sequences before spaces since space indexes use sequences.
+    local sequences = schema.sequences or {}
+    for sequence_name, sequence_schema in pairs(sequences) do
+        local ok, err = ddl_check.check_sequence(sequence_name, sequence_schema)
+        if not ok then
+            return nil, err
+        end
+
+        if box.sequence[sequence_name] ~= nil then
+            local current_schema = ddl_get.get_sequence_schema(sequence_name)
+            local _, err = ddl_compare.assert_equiv_sequence_schema(sequence_schema, current_schema)
+            if err ~= nil then
+                return nil, string.format(
+                    "Incompatible schema: sequences[%q] %s", sequence_name, err)
+            end
+        else
+            local ok, err = pcall(
+                ddl_set.create_sequence,
```

Comment:
At first I was surprised too, but it's actually a dry run, which seems appropriate and useful. (Just in case -- I haven't introduced this one, it already exists for spaces, but since dry-run spaces need sequences, we create them too.)

Reply:
Renamed a couple of internals from `_check` to `_check_and_dry_run`, since we already refactor them in this commit.